api/src/main/java/ca/bc/gov/educ/api/pen/services/controller/v1/PenServicesSagaController.java

```diff
@@ -32,12 +35,16 @@
   @Getter(PRIVATE)
   private final StudentMergeCompleteOrchestrator studentMergeCompleteOrchestrator;
 
+  @Getter(PRIVATE)
+  private final StudentDemergeCompleteOrchestrator studentDemergeCompleteOrchestrator;
+
   private static final SagaMapper sagaMapper = SagaMapper.mapper;
 
   @Autowired
-  public PenServicesSagaController(final SagaService sagaService, StudentMergeCompleteOrchestrator studentMergeCompleteOrchestrator) {
+  public PenServicesSagaController(final SagaService sagaService, StudentMergeCompleteOrchestrator studentMergeCompleteOrchestrator, StudentDemergeCompleteOrchestrator studentDemergeCompleteOrchestrator) {
```

Comment:
how about injecting List<orchestrator> to the Controller

Reply:
done.